docs/api.asciidoc

```diff
@@ -31,8 +31,10 @@ ElasticAPM.start(server_url: 'http://localhost:8200')
   options.  See <<configuration,Configuration>>.
 
 If you are using <<getting-started-rails,Ruby on Rails>> this is done
-automatically for you.
-If not see <<getting-started-rack,Getting started with Rack>>.
```

Comment:
What about something like:

```
If you are using <<getting-started-rails,Ruby on Rails>> this is done automatically for you.
If you choose not to require the `elastic_apm` gem and instead want to start the
agent and hook into Rails manually, see <<rails-start,hooking into Rails explicitly>>.
If you're not using Rails, see <<getting-started-rack,Getting started with Rack>>.
```

Reply:
The middle sentence was just a little tough for me to digest. Other than that, LGTM